### ISSUE TITLE
Do you really need the notation extension?

### ISSUE THREAD

**Issue by martinthomson** [quicwg/load-balancers#179]
> `x(A..B) y(C..B-len(x))` indicates that `x` can be of any length between `A` and `B`, and `y` can be of any length between `C` and `B` provided that `(len(x) + len(y))` does not exceed `B`.

This might be achieved as follows:

```
Outer {
  Inner (α..B),  <-- α = A + C
}
Inner {
  x(A..B),
  y(C..B),
}
```

**Comment by martinthomson** [comment]
I'll should also point out that `α` there is optional and you can also do:

```
Outer {
  Inner (..β),
}
Inner {
  x(A..B),
  y(C..D),
}
```

For any value of `β`, though presumably `β` is at least `max(B, D)`, it could reasonably be any value up to `B+D`.

**Comment by martinduke** [comment]
closed by #207
